### ISSUE TITLE
Specifying number of trials for marginal_effects

### ISSUE THREAD

**Issue by lukeholman** [paul-buerkner/brms#718]
Hi there,

When plotting a binomial brms model with `marginal_effects()` I get the message "Using the median number of trials by default if not specified otherwise."

Is there a way to get `marginal_effects()` to show the expected number of successes for a user-specified number of trials, instead of the default? I have read ?marginal_effects and googled, and cannot find any instructions on how to modify the number of trials. 

Also, perhaps it would be desirable to change the default so that brms assumes there are either 100 trials (so the y-axis can be interpreted as the percentage of successes), or 1 trial (proportion successes). The default of using the median number of trials is a bit confusing, because one needs to go and compute the median in order to interpret the plot. Instead of changing the default, which I know can cause trouble, one useful fix would be to change this message to state what the median number of trials is. For example it could read "Using the median number of trials (i.e. 127) by default if not specified otherwise."

Thanks!

**Comment by paul-buerkner** [comment]
Use argument `conditions` to set the number of trials manually. I agree though that using `1` as the default number of trials seems sensible. I will consider making this change.

**Comment by lukeholman** [comment]
Hi there,

Just reporting back that I tried and failed to set the number of trials using `conditions`. I tried the following, and neither of these options work (these are my best guesses from reading the docs): 

    marginal_effects(model, conditions = data.frame(trials = 1))
    marginal_effects(model, conditions = make_conditions(data.frame(trials = 1), vars = "trials"))

That is, I still get the message "Using the median number of trials by default if not specified otherwise", and the plot displays the median number of trials instead of the number I inputted here. I guess I am doing something wrong, but the function doesn't work the same wasy as most people would expect (i.e. the same as the familiar `newdata` argument to `predict()`), maybe one could expand the docs to illustrate how to specify the number of trials via `conditions`.
